src/components/CompiledOutput.js

```diff
@@ -45,8 +45,13 @@ export function CompiledOutput({
   const [showJSON, setShowJSON] = useState(false);
 
   let saveConfig = useCallback(() => {
-    let options = processOptions(config, debouncedPlugin);
-
+    let options;
+    try {
+      options = processOptions(config, debouncedPlugin);
+    } catch (error) {
+      console.error(error);
```

Comment:
For future UX, I think it'll be good to display this a popup error/message. Thoughts @penguingovernor?

Reply:
Wholeheartedly agree!